100-level-AsyncAPI-Intro/step2.md

```diff
@@ -0,0 +1,42 @@
+## Deep Dive into Various Components of AsyncAPI
+
+In the control room of Chan's spaceship, Eve is assisting Chan in setting up the communication system using AsyncAPI. They're working together to ensure they can maintain contact even when Chan is light-years away. Eve explains the different components of AsyncAPI as they program each part.
+
+## Understanding AsyncAPI Components through Chan and Eve's Programming Session
+
```

Comment:
I think you can remove these lines as I begin the module with a similar storyline, touch lightly on structure and lead to `deep dive of various components`, the rest fits in well and needs to modification for now as I see.